packages/config/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "@cms/config",
   "version": "0.1.0",
-  "private": true
+  "private": true,
+  "exports": {
+    ".": "./constants.ts"
```

Comment:
You want to explicitly state the interface for your workspaces here. `.` means you will be able to write `import {} from '@cms/config'`, the root of the package.

Reply:
Hmmm, didn't think about this one. Thank you.